### DISCUSSION TITLE
Overview: presenting a deprecated api

### DISCUSSION THREAD

**Discussion by plap979** [ReactiveX/rxjs#7520] [Report issues other than bug]
In the Overview, in the [Subject](https://rxjs.dev/guide/subject) page, we are presenting [multicasting APIs](https://rxjs.dev/guide/subject#multicasted-observables) which are now [deprecated](https://rxjs.dev/deprecations/multicasting).

So then I think the Subject page should present/teach about `connectable`, `connect` and `share`.

Tnx

**Comment by jakovljevic-mladen** [comment]
Yeah, this needs a thorough rewrite. However, it will take some time before V8 is stable and out, so we have time to do it. Are you interested in doing so?
